code/controllers/subsystem/processing/quirks.dm

```diff
@@ -32,6 +34,13 @@ PROCESSING_SUBSYSTEM_DEF(quirks)
 		list("Social Anxiety", "Mute"),
 	)
 
+	/// A list of quirks that cannot be used by specific species. Format: species = list(quirk1, quirk2)
+	var/static/list/species_quirk_blacklist = list(
+		/datum/species/plasmaman = list(
+			/datum/quirk/blooddeficiency,
+		),
```

Comment:
```suggestion
		SPECIES_PLASMAMAN = list(
			"Blood Deficiency",
		),
```
The prefs menu is unaware of the typepaths of the species as well as the quirk typepaths, so it's necessary to design the list like this.

Reply:
The pref menu should be changed to match this, not the other way around, I do not want breaking the prefs menu to be as easy as changing the name of a quirk, I think checking if a quirk is valid should be done on DM-side instead

Reply:
This really should be done in another PR, because I just wanted to help out Jolly with this one and I don't really intend on writing a refactor of prefs too lol